traffic_ops/app/db/admin.go

```diff
@@ -140,6 +159,7 @@ func parseDBConfig() error {
 		return errors.New("getting goose config: " + err.Error())
 	}
 
+	DBDriver = gooseCfg.Driver
```

Comment:
If the intention is to remove goose, then having type (GooseConfig), function (getGooseConfig), and variable (gooseCfg) with `goose` in their name is slightly misinformed.

Reply:
Renamed to `EnvConfig`, `getEnvironmentConfig`, and `envConfig` in 6168f7396e.